pom.xml

```diff
@@ -35,7 +35,7 @@
         <slf4j.version>2.6.18</slf4j.version>
         <cloudevents.version>2.4.2</cloudevents.version>
         <protobuf.version>3.21.10</protobuf.version>
-        <git.tag.name>uprotocol-core-api-1.5.6</git.tag.name>
+        <git.tag.name>main</git.tag.name>
```

Comment:
Can we create a release of up-core-api and use that version here to prevent compile-time errors in the Java SDK due to changes in existing proto fields merged into the main branch?

Reply:
Yes that will be on Friday